memory/libhugetlbfs.py

```diff
@@ -1,5 +1,23 @@
 #!/usr/bin/env python
 
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Santhosh G <santhog4@linux.vnet.ibm.com>
+#
+# Based on code by aganti@google.com (Ashwin Ganti)
+# copyright : 2008 Google, Inc.
+# https://github.com/autotest/autotest-client-tests/tree/master/libhugetlbfs
```

Comment:
Oups, Chris I mistakenly added you here. Also I forgot to check the latest changes, so `John Adamski` did a multiple contributions and as he claimed the authorship he should be listed along with the year of his first contribution.